src/libstd/Cargo.toml

```diff
@@ -24,6 +24,7 @@ compiler_builtins = { version = "0.1.16" }
 profiler_builtins = { path = "../libprofiler_builtins", optional = true }
 unwind = { path = "../libunwind" }
 hashbrown = { version = "0.4.0", features = ['rustc-dep-of-std'] }
+getrandom = "0.1"
```

Comment:
Caveat: this won't work on bare WASM since `getrandom` currently requires a feature flag to enable either `stdweb` or `wasm_bindgen`. There was mention of making `wasm_bindgen` the default; we might want to do this first?

Reply:
Personally I would prefer if a separate target will be introduced and `wasm32-unknown-wunknown` will stay assumptions-free regarding executor enviroment.

Reply:
IIUC without enabled `stdweb` or `wasm_bindgen` features outside of WASI and Emscripten it will use a dummy impl, so it will work in the same way as it does today by using a constant seed value.